app/views/layouts/application.html.erb

```diff
@@ -15,6 +15,7 @@
     <div class="container">
       <%= yield %>
       <%= render "layouts/footer" %>
+      <%= debug(params) if Rails.env.development? %>
```

Comment:
I could see this being convenient. You might also use `params.inspect` for better legibility